cterasdk/edge/network.py

```diff
@@ -176,3 +177,57 @@ def iperf(self, address, port=5201, threads=1, protocol=IPProtocol.TCP, directio
             return task.result.res
         except TaskError as error:
             return error.task.result.res
+
+    def get_static_routes(self):
+        """
+        Get all Static Routes
+        """
+        return self._gateway.get('/config/network/static_routes')
+
+    def set_static_route(self, source_ip, destination_ip, destination_mask):
```

Comment:
Should be changed to "add_static_route" and "remove_static_route". "Set" suggests replacing the existing configuration alltogether. add/remove is consistent with other areas with similar functionality in the SDK.